scripts/repl.js

```diff
@@ -28,12 +28,12 @@ if (process.env.BS_PLAYGROUND == null) {
 var playground = process.env.BS_PLAYGROUND
 
 function prepare() {
-    e(`hash hash js_of_ocaml 2>/dev/null || { echo >&2 "js_of_ocaml not found on path. Please install version 2.8.4 (although not with the buckelscript switch) and put it on your path."; exit 1; }
-`)
+//     e(`hash hash js_of_ocaml 2>/dev/null || { echo >&2 "js_of_ocaml not found on path. Please install version 2.8.4 (although not with the buckelscript switch) and put it on your path."; exit 1; }
+// `)
 
-    e(`hash ocp-ocamlres 2>/dev/null || { echo >&2 "ocp-ocamlres not installed. Please install: opam install ocp-ocamlres"; exit 1; }`)
+//     e(`hash ocp-ocamlres 2>/dev/null || { echo >&2 "ocp-ocamlres not installed. Please install: opam install ocp-ocamlres"; exit 1; }`)
 
-    e(`hash camlp4 2>/dev/null || { echo >&2 "camlp4 not installed. Please install: opam install camlp4"; exit 1; }`)
+//     e(`hash camlp4 2>/dev/null || { echo >&2 "camlp4 not installed. Please install: opam install camlp4"; exit 1; }`)
 
     // require('../scripts/release').run()
```

Comment:
why these lines commented?

Reply:
@bobzhang this was commented in your commit at https://github.com/BuckleScript/bucklescript/commit/740f0bfc98d310a10863b3fc0f8a5db211bf5e69
I think it was because the JSX PPX wasn't compiling.